core/src/test/scala/integration/kafka/api/AdminClientWithPoliciesIntegrationTest.scala

```diff
@@ -81,7 +81,7 @@ class AdminClientWithPoliciesIntegrationTest extends KafkaServerTestHarness with
   }
 
   @ParameterizedTest
-  @ValueSource(strings = Array("zk", "kraft"))
+  @ValueSource(strings = Array("kraft"))
```

Comment:
Further remove zk test in `AdminClientWithPoliciesIntegrationTest.scala` as this test class is somehow related to `PlaintextAdminIntegrationTest.scala` as AdminClientWithPoliciesIntegrationTest.testInvalidAlterConfigs invoke PlaintextAdminIntegrationTest#checkInvalidAlterConfigs